mizar/daemon/app.py

```diff
@@ -119,6 +125,23 @@ def init(benchmark=False):
     output = r.stdout.read().decode().strip()
     logging.info("Running load-transit-xdp: {}".format(output))
 
+    if os.getenv('FEATUREGATE_OFFLOAD_XDP', 'false').lower() in ('true', '1'):
+        if default_itf in support_offload_xdp_itf_names():
+            config = {
```

Comment:
Does XDP offload support debug mode programs?

Reply:
Due to the size limitation of instructions, we remove the debug codes in our offload program.

Reply:
understood.